### DISCUSSION TITLE
Custom smtp server seems to not work

### DISCUSSION THREAD

**Discussion by Olyno** [supabase/supabase#451] [Questions]
Hi 👋🏻 

So I tried the custom smtp service by linking one of my gmail address and supabase. I did all (like all) the things to make it works at my side (like allow less secure apps, allow imap, pop....) but I still have an error when it must to send an email. Unfortunetely, the error says nothing except "There is an error when sending the email".

Any idea of what is going on? And/Or how could I fix it?

**Comment by kiwicopple** [comment]
Hey @Olyno - did you test the SMTP settings with another service? 

Perhaps we can add a button for sending test emails? and also we are working on exposing the GoTrue logs which should make this easier for you to debug.

**Reply by Olyno** [reply]
Exposing GoTrue logs could be benefic. It could able us to report you some errors, or simply fix them if the error comes from a bad setup

**Reply by kiwicopple** [reply]
OK cool - we're working on a logging solution now ...
